src/main/java/com/exasol/matcher/ResultSetStructureMatcher.java

```diff
@@ -56,6 +54,8 @@ private ResultSetStructureMatcher(final Builder builder) {
         for (final Object expectedCellValue : expectedRow) {
             if (expectedCellValue instanceof Matcher<?>) {
                 rowOfMatchers.add(castToMatcher(expectedCellValue));
+            } else if (expectedCellValue == null) {
+                rowOfMatchers.add(is(nullValue()));
```

Comment:
```suggestion
                rowOfMatchers.add(nullValue());
```
Should be sufficient as far as I remember.

Reply:
:+1: